tool/python/singa/layer.py

```diff
@@ -228,21 +228,20 @@ def __init__(self, size=0, **kwargs):
 
 class Activation(Layer):
 
-    def __init__(self, activation='stanh', topk=1):
+    def __init__(self, activation='stanh'):
         '''
         required
-          activation = (string)
-        optional
-          topk       = (int)  // the number of results
+          activation = (string) // relu, sigmoid, tanh, stanh, softmax.
         '''
 
+        if activation == 'tanh':
```

Comment:
In short,
- singa supports sigmoid, relu, tanh for both cpu and gpu by ActivationLayer and cudnnActivationLayer, respectively.
- It supports stanh by STanhLayer for cpu.
- We will update it for gpu soon.

Thus, simply we remove lines 237 and 238, right?